### PR TITLE
P0   alert & button

### PR DESCRIPTION
**Pull request by saicharan-gouru**
please review my alert and button components. I'm attaching the demo below.

alert and button

### CODE REVIEW

File: Button/button.css [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+:root {
+    --primary-red: #DC2626;
+    --primary-green: #34D399;
+    --primary-yellow: #FBBF24;
+    --primary-blue: #2563EB;
+    --secondary-red: #F87171;
+    --secondary-green: #A7F3D0;
+    --secondary-yellow: #FDE68A;
+    --secondary-blue: #60A5FA;
+    --white: white;
```
Comment (Naveenchandar) [inline Button/button.css]: Suggestion:
Please use hex code values for consistency.

**Review by Aditi0505 — APPROVED** [review]
Looks good to me. You can make small changes for maintaining consistency.

**Review by Naveenchandar — APPROVED** [review]
Looks good

**Comment by saicharan-gouru** [comment]
> Looks good to me. You can make small changes for maintaining consistency.

Thanks for the review. please mention those changes to be made, I won't repeat them again.

File: Button/button.css (hunk 1 above)
Comment (saicharan-gouru) [reply Button/button.css]: Thanks for the review. I have made the changes.

**Review by maitrakhatri — APPROVED** [review]
looks perfect to me

**Pull request merged**